### ISSUE TITLE
GIVdrop - Claim your GIV page text fix

### ISSUE THREAD

**Issue by laurenluz** [Giveth/GIVeconomy#303]
Change "Join the giving economy." to "Let's Build the Future of Giving, together."

![image.png](https://images.zenhubusercontent.com/5fdfac16a58523a866ddb68d/b69d26dc-bb17-45c3-9fea-4fbd1aa2ab4c)

**Comment by laurenluz** [comment]
Thsi still needs to updated @pkretzschmar maybe it didn't get merged?

![image](https://user-images.githubusercontent.com/75490971/146579620-ce9c6eb2-fdd1-43c1-ac0a-e6a33c7774c1.png)

**Comment by GriffGreen** [comment]
This needs to be redone?

**Comment by divine-comedian** [comment]
duplicate of #375

**Comment by divine-comedian** [comment]
pending on https://github.com/Giveth/liquidity-mining-dapp/pull/73
